## Projeto estatístico com dados de times nacionais de futebol

Este projeto realiza uma análise estatística simples usando R. As etapas incluem:
1. Cálculo de probabilidades (condicionais e incondicionais);
2. Construção e interpretação de intervalos de confiança;
3. Testes de hipóteses (comparando média populacional e média amostral);
4. Análise de regressão linear (simples ou múltipla).

Os dados foram extraídos de um conjunto de jogadores de seleções nacionais.

##### 1. Cálculo de probabilidades condicionais e incondicionais

Foram selecionadas duas variáveis para calcular probabilidades condicionais e incondicionais:
- `nationality`: nacionalidade dos jogadores.
- `age`: faixa etária dos jogadores.

O código a seguir importa os dados e calcula probabilidades simples e condicionais.

In [1]:
# Carregando os dados
dados <- read.csv("../data/Anexo_Projeto_fifa_world_national_teams_versao_oficial_20241virgula.csv", sep=",", stringsAsFactors = FALSE)

# Criando faixas etárias
dados$faixa_etaria <- cut(dados$age, breaks=c(15, 20, 25, 30, 35, 40), right=FALSE, 
                          labels=c("15-19", "20-24", "25-29", "30-34", "35-39"))

# Calculando probabilidades incondicionais
prop_nacionalidades <- prop.table(table(dados$nationality))
prop_faixas_etarias <- prop.table(table(dados$faixa_etaria))

# Probabilidades condicionais
brasileiros <- subset(dados, nationality == "Brazil")
prop_faixa_brasileiros <- prop.table(table(brasileiros$faixa_etaria))

# Exibindo resultados
prop_nacionalidades
prop_faixas_etarias
prop_faixa_brasileiros



                              Argentina           Australia             Austria 
        0.019498607         0.023676880         0.018105850         0.022284123 
            Belgium              Brazil            Bulgaria            Cameroon 
        0.026462396         0.032033426         0.004178273         0.023676880 
             Canada               Chile            Colombia      Czech Republic 
        0.026462396         0.013927577         0.016713092         0.027855153 
            Denmark             Ecuador               Egypt             England 
        0.032033426         0.008356546         0.009749304         0.032033426 
            Finland              France             Germany              Greece 
        0.026462396         0.032033426         0.032033426         0.029247911 
            Hungary             Iceland               Italy              Mexico 
        0.011142061         0.018105850         0.030640669         0.013927577 
        Netherlands        


     15-19      20-24      25-29      30-34      35-39 
0.01278409 0.29971591 0.46875000 0.20312500 0.01562500 


    15-19     20-24     25-29     30-34     35-39 
0.0000000 0.0000000 0.3478261 0.4347826 0.2173913 

##### 2. Intervalos de confiança e sua interpretação para pelo menos duas variáveis selecionadas

Para calcular intervalos de confiança e interpretá-los, serão selecionadas as seguintes variáveis:

- overall_rating: avaliação geral dos jogadores.
- wage_euro: salário em euros.

In [2]:
# Carregando o pacote necessário
if (!require("dplyr")) install.packages("dplyr")
if (!require("DescTools")) install.packages("DescTools")

library(dplyr)
library(DescTools)

Carregando pacotes exigidos: dplyr


Anexando pacote: 'dplyr'


Os seguintes objetos são mascarados por 'package:stats':

    filter, lag


Os seguintes objetos são mascarados por 'package:base':

    intersect, setdiff, setequal, union


Carregando pacotes exigidos: DescTools

Warning message in library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE, :
"não há nenhum pacote chamado 'DescTools'"
Instalando pacote em 'C:/Users/Marcel Albuquerque/AppData/Local/R/win-library/4.4'
(como 'lib' não foi especificado)

instalando as dependências 'proxy', 'rootSolve', 'e1071', 'lmom', 'mvtnorm', 'expm', 'Exact', 'gld' também




pacote 'proxy' desempacotado com sucesso e somas MD5 verificadas
pacote 'rootSolve' desempacotado com sucesso e somas MD5 verificadas
pacote 'e1071' desempacotado com sucesso e somas MD5 verificadas
pacote 'lmom' desempacotado com sucesso e somas MD5 verificadas
pacote 'mvtnorm' desempacotado com sucesso e somas MD5 verificadas
pacote 'expm' desempacotado com sucesso e somas MD5 verificadas
pacote 'Exact' desempacotado com sucesso e somas MD5 verificadas
pacote 'gld' desempacotado com sucesso e somas MD5 verificadas
pacote 'DescTools' desempacotado com sucesso e somas MD5 verificadas

Os pacotes binários baixados estão em
	C:\Windows\Temp\Rtmpgp4UAl\downloaded_packages


In [3]:
# Removendo valores NA
dados <- na.omit(dados)

# Selecionando as variáveis
overall_rating <- dados$overall_rating
wage_euro <- dados$wage_euro

# Intervalo de confiança para overall_rating (95%)
ic_overall <- MeanCI(overall_rating, conf.level = 0.95)
print("Intervalo de confiança para overall_rating (95%):")
print(ic_overall)

# Intervalo de confiança para wage_euro (95%)
ic_wage <- MeanCI(wage_euro, conf.level = 0.95)
print("Intervalo de confiança para wage_euro (95%):")
print(ic_wage)

[1] "Intervalo de confiança para overall_rating (95%):"
    mean   lwr.ci   upr.ci 
76.50426 76.05738 76.95114 
[1] "Intervalo de confiança para wage_euro (95%):"
    mean   lwr.ci   upr.ci 
49998.58 45140.11 54857.05 


Resultados e Interpretação
> Para overall_rating

    Média amostral: 76,50
    Intervalo de confiança (95%): [76,06; 76,95]
>
> Temos 95% de confiança de que a média da avaliação geral dos jogadores na população está entre 76,06 e 76,95. Isso sugere que, de forma consistente, a qualidade média dos jogadores é bastante alta.

> Para wage_euro (salário em euros):

    Média amostral: 49.998,58 euros
    Intervalo de confiança (95%): [45.140,11; 54.857,05]

> Com 95% de confiança, pode-se dizer que o salário médio populacional dos jogadores está entre 45.140,11 e 54.857,05 euros. Isso demonstra que os jogadores das seleções nacionais tendem a ter salários expressivos no cenário global.

##### 3. Testes de hipóteses e interpretação para pelo menos duas variáveis selecionadas. As hipóteses devem avaliar se a média populacional da referida variável pode ser considerada igual à média amostral arredondada para o inteiro. 

Para realizar testes de hipóteses, serão selecionadas as mesmas variáveis do passo anterior: overall_rating e wage_euro. 

In [4]:
# Removendo valores NA
dados <- na.omit(dados)

# Selecionando as variáveis
overall_rating <- dados$overall_rating
wage_euro <- dados$wage_euro

# Hipótese 1: Teste para overall_rating
media_overall_arredondada <- round(mean(overall_rating))
teste_overall <- t.test(overall_rating, mu = media_overall_arredondada, conf.level = 0.95)

print("Teste de hipótese para overall_rating:")
print(teste_overall)

# Hipótese 2: Teste para wage_euro
media_wage_arredondada <- round(mean(wage_euro))
teste_wage <- t.test(wage_euro, mu = media_wage_arredondada, conf.level = 0.95)

print("Teste de hipótese para wage_euro:")
print(teste_wage)

[1] "Teste de hipótese para overall_rating:"

	One Sample t-test

data:  overall_rating
t = -2.178, df = 703, p-value = 0.02974
alternative hypothesis: true mean is not equal to 77
95 percent confidence interval:
 76.05738 76.95114
sample estimates:
mean of x 
 76.50426 

[1] "Teste de hipótese para wage_euro:"

	One Sample t-test

data:  wage_euro
t = -0.00016991, df = 703, p-value = 0.9999
alternative hypothesis: true mean is not equal to 49999
95 percent confidence interval:
 45140.11 54857.05
sample estimates:
mean of x 
 49998.58 



Resultados e Interpretação:
> 1. Teste para overall_rating
Hipóteses:

    H0H0​: A média populacional (μμ) é igual a 77.
    H1H1​: A média populacional (μμ) é diferente de 77.

Resultados:

    t=−2.178t=−2.178
    p-valor = 0.02974
    Intervalo de confiança de 95%: [76.06; 76.95]
    Média amostral: 76.50

> Há uma diferença significativa entre a média populacional e a média amostral arredondada.

> 2. Teste para wage_euro
Hipóteses:

    H0H0​: A média populacional (μμ) é igual a 49.999 euros.
    H1H1​: A média populacional (μμ) é diferente de 49.999 euros.

Resultados:

    t=−0.00016991t=−0.00016991
    p-valor = 0.9999
    Intervalo de confiança de 95%: [45.140,11; 54.857,05]
    Média amostral: 49.998,58 euros

> Não há evidências para rejeitar que a média populacional seja igual à média amostral arredondada.